<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Skyplot</class>
 <widget class="QDialog" name="Skyplot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skyplot</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="GAO_cek">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Galileo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="GPS_cek">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GPS</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="BDS_cek">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BD2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="GLS_cek">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GLONASS</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
